clients/drcachesim/tracer/instru.h

```diff
@@ -221,10 +223,15 @@ class instru_t {
     // This is a per-buffer-writeout header.
     virtual int
     append_unit_header(byte *buf_ptr, thread_id_t tid, ptr_int_t window) = 0;
+    // If the entry at buf_ptr is not a timestamp, returns false.
+    // Else, if the timestamp value is < min_timestamp, replaces it with a
+    // new timestamp (ignoring the frozen timestamp).  Returns true either way.
```

Comment:
I didn't like the idea of a bunch of threads with perfectly identical timestamps: that was my logic as it looks artificial.  Do you think we should use the minimum anyway?

Reply:
Since we are starting tracing for all threads together, it seemed fine to me if all threads had perfectly identical timestamps for their start. Also makes explaining the timestamp entry easier, instead of "it's the timestamp of the first entry for all non-first buffers"